src/transitions/HorizontalSlide.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <transition appear
+        :enter-active-class="'animated ' + (isRTL ? 'slideInRight' : 'slideInLeft' )"
+        :leave-active-class="'animated ' + (isRTL ? 'slideOutRight' : 'slideOutLeft') "
+        v-bind="$attrs"
+        v-on="$listeners">
+        <slot/>
+    </transition>
+</template>
+
+<script>
+export default {
+    name: 'HorizontalSlide',
+
+    props:{
+        isRTL:{
```

Comment:
let's rename this to rtl

Reply:
ok i will change it,
and will add the HorizontalFade as well